src/WordPress/Hooks.php

```diff
@@ -250,4 +252,11 @@ public function http2ServerPushInit()
     {
         HTTP2ServerPush::init();
     }
+
+    public function getCloudflareRequestJSON()
+    {
+        if ($_GET['action'] === 'cloudflare_proxy') {
```

Comment:
Let's put `cloudflare_proxy` into a global variable in Hooks.php. It is being used in [index.php](https://github.com/cloudflare/Cloudflare-WordPress/blob/master/index.php#L49) as well. 

`const CLOUDFLARE_PROXY = "cloudflare_proxy"`